libraries/lib-numeric-formats/ProjectTempoListenerInterface.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <memory>
+#include <optional>
+
+class NUMERIC_FORMATS_API ProjectTempoListenerInterface
+{
+public:
+   virtual ~ProjectTempoListenerInterface();
+   virtual void OnProjectTempoChange(
+      const std::optional<double>& oldTempo, double newTempo) = 0;
```

Comment:
As I said before:  this entire class could be eliminated, and I stead you can write

`using ProjectTempoListener = std::function<…>;`

Reply:
… if in fact there remains any need to define this interface, but I doubt even that